src/MockedRequest.ts

```diff
@@ -153,17 +153,32 @@ export class MockedRequest {
         )
     }
 
+    private wasInvokedWithContractAndAlsoPassed(
+        passedInContract: Contract
+    ): boolean {
+        if (passedInContract && this.contract) {
+            return true
+        }
+        return false
```

Comment:
Would this work?
```suggestion
        return passedInContract && this.contract
```

Reply:
This would return `this.contract`. A test in REPL:

```
> var passedInContract = {}
undefined
> var contract = {}
undefined
> passedInContract && contract
{}
```